src/components/InputForm.tsx

```diff
@@ -0,0 +1,66 @@
+import styled from 'styled-components';
+import { Card } from '../types/card';
+import InputField from './InputField';
+import INPUT_TYPE_CATEGORIES from '../constants/inputType';
+
+interface Props {
+  cardInfo: Card;
+  handleInput: (value: Card) => void;
+}
+
+export default function InputForm({ cardInfo, handleInput }: Props) {
+  const handleCardNumberInput = (value: { [key: string]: string }) => {
```

Comment:
```suggestion
  const handleCardNumberInput = (value: Record<string, string>) => {
```

으로 수정할 수 있을 것 같아요! 혹은 key의 타입은 더 좁혀줄 수 있을것도 같구요!

Reply:
Record 유틸 함수에 대해 잘 알지 못했는데 이런 방법이 있군요! 감사합니다!